spec/unit/block_score/candidate_spec.rb

```diff
@@ -1,137 +1,121 @@
 
-      it { is_expected.not_to be_persisted }
-      its(:class) { should be BlockScore::Candidate }
-    end
-
-    describe '.create' do
-      subject(:candidate) { BlockScore::Candidate.create(attributes_for(:candidate)) }
-
-      it { is_expected.to be_persisted }
-      its(:class) { should be BlockScore::Candidate }
-    end
+    it { is_expected.not_to be_persisted }
+    it { is_expected.to be_an_instance_of described_class }
+  end
 
-    describe '.retrieve' do
-      let(:candidate_id)  { create(:candidate).id }
-      subject(:candidate) { BlockScore::Candidate.retrieve(candidate_id) }
+  describe '.create' do
+    subject(:candidate) { described_class.create(attributes_for(:candidate)) }
 
-      it { is_expected.to be_persisted }
-      its(:name_first) { is_expected.not_to be_empty }
-      its(:class) { should be BlockScore::Candidate }
-    end
+    it { is_expected.to be_persisted }
+    it { is_expected.to be_an_instance_of described_class }
+  end
 
-    # NB: At the time of writing, the testing API does not reset. Making this
-    #     test less then perfect. To work around the near certain availability
-    #     of historical candidates from other tests known candidates are created
-    #     and checked. This has the side effect of being susceptible to race
-    #     conditions if tested against the live API in parallel.
-    describe '.all' do
-      let(:uniq_token_one) { '05816347d2234ef4a92465e9784c7ce1' }
-      let(:uniq_token_two) { 'b4cd5fb645788c3b37c11407b83d7741' }
-      subject(:candidates) { BlockScore::Candidate.all }
+  describe '.retrieve' do
+    let(:candidate_id) { create(:candidate).id }
+    subject(:candidate) { described_class.retrieve(candidate_id) }
 
-      before do
-        create(:candidate, name_first: 'John', name_last: uniq_token_one)
-        create(:candidate, name_first: 'John', name_last: uniq_token_two)
-      end
+    it { is_expected.to be_persisted }
+    its(:name_first) { is_expected.not_to be_empty }
+    it { is_expected.to be_an_instance_of described_class }
+  end
 
-      it { is_expected.not_to be_empty }
-      it { expect(candidates[0].name_last).to eql uniq_token_two }
-      it { expect(candidates[1].name_last).to eql uniq_token_one }
+  # NB: At the time of writing, the testing API does not reset. Making this
+  #     test less then perfect. To work around the near certain availability
+  #     of historical candidates from other tests known candidates are created
+  #     and checked. This has the side effect of being susceptible to race
+  #     conditions if tested against the live API in parallel.
+  describe '.all' do
+    let(:uniq_token_one) { '05816347d2234ef4a92465e9784c7ce1' }
+    let(:uniq_token_two) { 'b4cd5fb645788c3b37c11407b83d7741' }
+    subject(:candidates) { described_class.all }
+
+    before do
+      create(:candidate, name_first: 'John', name_last: uniq_token_one)
+      create(:candidate, name_first: 'John', name_last: uniq_token_two)
```

Comment:
I think creating objects would be better done in `let!` blocks rather than a `before` block. Their purpose is to create specific objects to support the test and should be preferred over the more general `before` blocks.